tests/console/zypper_extend.pm

```diff
@@ -65,6 +79,39 @@ sub run {
     #Remove star, don't ask for permission:
     zypper_call 'rm star';
 
+    #Install specific version of a package
+    my $version = script_output q[zypper se -s star |grep " star " | awk 'END {print $6}'];
+    zypper_call 'in -f star-$version';
+
+    #Install and remove package combination
+    zypper_call 'in vim -star';
+
+    #Cleaning up dependencies of removed packages
+    zypper_call 'rm --clean-deps vim';
+
+    #Add a repository
+    zypper_call 'ar -p 90 -f --no-gpgcheck http://ftp.gwdg.de/pub/linux/misc/packman/suse/openSUSE_Tumbleweed/ packman';
+    assert_script_run("zypper lr | grep packman");
+
+    #Install package from a disabled repository
+    zypper_call 'mr -d packman';
+    zypper_call '--plus-content packman install rpmkey-packman';
+
+    #Prioritize a repository
+    zypper_call 'mr -e -p 20 packman';
+
+    #Rename a repository
+    zypper_call 'renamerepo "packman" Packman';
+
+    #Remove a repository
+    zypper_call 'rr packman';
```

Comment:
you  just renamed the repo to *Packman* just the command before - the repo you attempt to remove no longer exists

Reply:
Yes, you are right! I fix this here : https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10518